Initium-Odp/war/odp/ajax_autofix.jsp

```diff
@@ -0,0 +1,30 @@
+<%@ taglib uri='http://java.sun.com/jsp/jstl/core' prefix='c'%>
```

Comment:
This is the older way of adding a page. Now pages go in war/web-inf/odppages and must have an accompanying controller to go with it. In your case the controller will be basically empty, but it's a good chance to try your hand at it.